cli/cli.go

```diff
@@ -576,6 +584,24 @@ func (ec *ExecutionContext) Spin(message string) {
 	}
 }
 
+// loadEnvfile loads .env file
+func (ec *ExecutionContext) loadEnvfile() error {
+	ec.Envfile += ".env"
```

Comment:
User should be able to give the complete file path instead of just the name and us adding the extension. And `.env` should work by default.

Reply:
Accounted for explicit `.env` filename provided by flag. I've also added an INFO level acknowledgement in case the `.env` file is read.